docs/spec_draft.md

```diff
@@ -1268,6 +1269,89 @@ More formally, `result[i0, ..., iR-1]` is equal to:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.reduce
+
+### Semantics
+
+Performs a reduction function `body` over the `inputs` and `init_values` along
```

Comment:
I don't think it's super idiomatic to say "perform a function over inputs". "Apply a function to inputs" is one of the more idiomatic options.